vision/getting-started/image_editing.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # @title Authenticate your notebook environment (Colab only)
</pre><p>Can you move all of these @title/@markdown to the markdown cells above the code cell?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/489/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I feel like that defeats the purpose of the markdown feature? otherwise we would just use a text cell. This is following the same practice as Model Garden notebooks that have the @title/markdown in the code cells. Example: https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/model_garden/model_garden_pytorch_llama2_peft_finetuning.ipynb

Reply:
My suggestion would be to use text cells for the markdown. Those annotations only work in Colab, but not when running locally. This makes it easier to follow in other tools.